code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -50,7 +48,7 @@
 	. = ..()
 	buildstage = building
 	if(buildstage)
```

Comment:
Shouldn't this be `buildstage == 3`?
There shouldn't be a case currently where it spawns half build though

Reply:
Maybe? I was originally going to fix a lot of this stuff, but refactoring intercomm wall building was quickly turning into something that was way more than I wanted to deal with, so I scaled it back to just the performance aspect of this and let it be.